lib/matplotlib/axis.py

```diff
@@ -604,6 +604,14 @@ class Ticker(object):
     locator = None
     formatter = None
 
+    def __init__(self, ticker=None):
```

Comment:
I have no problem doing it that way too. Functionally it won't make a difference what the name of the method is.